### PR TITLE
Modifications to display tests and run them in the Mediawiki UI

### PR DESCRIPTION
**Pull request by arthurpsmith**
I've tried this particularly on Z144, Z148, Z61, and also some random non-functions - hopefully it doesn't break something I haven't checked!

### CODE REVIEW

File: includes/AbstractTextAPI.php [hunk 1]
```diff
@@ -62,10 +82,6 @@ public function execute() {
 		);
 	}
 
-	public function mustBePosted() {
-		return false;
```
Comment (vrandezo) [inline includes/AbstractTextAPI.php:66]: Why that?

File: includes/AbstractTextContent.php [hunk 2]
```diff
@@ -56,7 +56,9 @@ public function getParserOutput(
     $data_arg = escapeshellarg($data);
 		$title_name = $title->getText();
 
-    $cmd = "node $script_path --lang:$lang --http $murl 'Z36($title_name)'";
+//    $cmd = "node $script_path --lang:$lang --http $murl 'Z36($title_name)'";
+# Somehow this was nont working with the --http setting
```
Comment (vrandezo) [inline includes/AbstractTextContent.php:60]:
```suggestion
# Somehow this was not working with the --http setting
```

File: includes/AbstractTextAPI.php (hunk 1 above)
Comment (arthurpsmith) [reply includes/AbstractTextAPI.php:66]: Seems to be a problem with the git diff function here - or maybe I modified indentation? It's still there just above in the code, lines 46-48

**Comment by arthurpsmith** [comment]
Uh, see above, it seems not to like your account here for some reason?!

**Comment by vrandezo** [comment]
@googlebot rescan

**Comment by vrandezo** [comment]
Fixed now, thanks.

**Comment by vrandezo** [comment]
Still curious about the mustBePosted setting - shouldn't it be OK to have it go via a GET request?

**Comment by arthurpsmith** [comment]
> Still curious about the mustBePosted setting - shouldn't it be OK to have it go via a GET request?

Yes, I didn't change it, git just thinks it changed I think because I moved the bulk of the first function to the end. Check the new code, that same code is still there in lines 46-48.

**Comment by vrandezo** [comment]
Ah, sorry, missed that!

Thank you so much, this is really, really cool!

**Pull request merged**